README.md

```diff
@@ -70,19 +72,48 @@ The Kata Containers project is governed according to the ["four opens"](https://
+e.g. https://github.com/kata-containers/community/issues/403. Committers have write access to the Kata Containers repos on GitHub, which
+gives the ability to approve PRs, trigger the CI and merge PRs.
+
+One of the requirements to be a committer is that you are an active contributor to the project as adjudged by the above criteria. Assessing the list of active contributors happens twice a year,
+lining up with the Architecture Committee election cycle. At that time, people who are in the kata-containers-committer team will also be reviewed, and a list of people, who are on the team,
+but who haven't been an active contributor in the last 12 months will be created and shared with the Architecture Committee and community.
+After a short review period, to allow time to check for any errors, inactive team members will be removed.
+
+> [!Note]
+> See [issue #413](https://github.com/kata-containers/community/issues/413) for a potential change in how active contribution is assessed.
+
+### Admin
+
+Kata Containers Admins (as defined by the [kata-containers-admin team](https://github.com/orgs/kata-containers/teams/kata-containers-admin) have admin access to
+the kata-containers repo, allowing them to do actions like, change the branch protection rules for repositories, delete a repository and manage the access of others.
+The Admin group is intentionally kept small, however, individuals can be granted temporary admin access to carry out tasks, like creating a secret that is used in a particular CI infrastructure.
+The Admin list is reviewed and updated after each Architecture Committee election and typically contains:
+- The Architecture Committee
+- Optionally, some specific people that the Architecture Committee agree on adding for a specific purpose (e.g. to manage the CI)
+
+### Owner
+
+GitHub organization owners have complete admin access to the organization, and therefore this group is limited to a small number of individuals, for security reasons.
+The owners list is reviewed and updated after each Architecture Committee election and contains:
+- The Community Manager and one, or more extra people from the `OpenInfra Foundation` for redundancy and vacation cover
+- The Architecture Committee
+- Optionally, some specific people that the Architecture Committee agree on adding for a specific purpose (e.g. to help with repo/CI migration)
 
 ## Architecture Committee
 
-The Architecture Committee is responsible for architectural decisions, including standardization, and making final decisions if Maintainers disagree. It is comprised of 7 members, who are elected by contributors.
+The Architecture Committee is responsible for architectural decisions, including standardization, and making final decisions if Committers disagree. It is comprised of 7 members, who are elected by contributors.
```

Comment:
In general in this doc I would use:
 * "Contributor" (capitalized) to refer to the specific Contributor developer role.
 * "developer" to refer broadly to the 4 developer roles.

```suggestion
The Architecture Committee is responsible for architectural decisions, including standardization, and making final decisions if Committers disagree. It is comprised of 7 members, who are elected by Contributors.
```

Reply:
I think I've updated based on your suggestions...